SimpleWAWS/Code/SiteNameGenerator.cs

```diff
@@ -10,5 +10,10 @@ public static string GenerateName()
             //first 4 parts of a guid
             return string.Format(CultureInfo.InvariantCulture, "{0}-0ee0-4-231-b9ee", Guid.NewGuid().ToString().Substring(0, 8));
         }
+        public static string GenerateVmName()
+        {
+            //first 4 parts of a guid
+            return string.Format(CultureInfo.InvariantCulture, "a{0}-0ee0-4-231-b9ee", Guid.NewGuid().ToString().Substring(0, 7));
```

Comment:
This is actually the dns name. There is a specific regex check for it. Which this format works. I'll update the function name , and see if I can get the regex added in the comment